http-server/src/server.rs

```diff
@@ -113,26 +112,19 @@ pub struct Server {
 	/// Local address
 	local_addr: Option<SocketAddr>,
 	/// Registered methods.
-	root: RpcModule,
+	methods: Methods,
 	/// Max request body size.
 	max_request_body_size: u32,
 	/// Access control
 	access_control: AccessControl,
 }
 
 impl Server {
-	/// Register a new RPC method, which responds with a given callback.
-	pub fn register_method<F, R>(&mut self, method_name: &'static str, callback: F) -> Result<(), Error>
```

Comment:
random idea until we have a proper builder for this; is keep this method but internally create a RpcModule<()> and append it the the methods?

Reply:
I think this will be annoying to implement nvm